configs/sites/s4/packages.yaml

```diff
@@ -166,8 +173,8 @@ packages:
       prefix: /usr
   qt:
     externals:
-    - spec: qt@5.15.3
-      prefix: /data/prod/jedi/spack-stack/qt-5.15.3
+    - spec: qt@5.9.7
+      prefix: /usr
```

Comment:
This change looks like a switch from using a spack-stack built qt library to one provided on the S4 system. Does this mean that somewhere there should be a change where the spack-stack qt build and install can is shut off?

Reply:
Correct, the sysadmins installed the missing packages and now we can use that version.